Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -1752,6 +1849,7 @@ BoardInit (
     PcieRpPwrRstInit ();
     RtcInit ();
     PlatformFeaturesInit ();
+    SetSSCDisable();

```

Comment:
SSC should not be disabled for all boards.
I think it needs to be enabled for all current CRB boards and UP2.
If you want to do it for your board, then it needs to be in CFGDATA so that you can set it in your delta file.

Reply:
would you be able to review the commit here: https://github.com/andreyv1978/slimbootloader/pull/1

Reply:
Sure, let me review that.

Reply:
If you happy with a final changes (from Monday), can you guide me the procedure of merging to the main branch